### PR TITLE
FontRendering/PlottingGL: Bug fixes to prevent errors due to string w…

### PR DESCRIPTION
**Pull request by ResearchDaniel**
…ith space in font rendering and empty axis in parallel coordinates

### CODE REVIEW

File: modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp [hunk 1]
```diff
@@ -395,6 +395,10 @@ void ParallelCoordinates::createOrUpdateProperties() {
 }
 
 void ParallelCoordinates::buildLineMesh() {
+    if (axes_.empty()) {
```
Comment (petersteneteg) [inline modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp]: seems strange to handle this here, should be handled in process

**Review by petersteneteg — CHANGES_REQUESTED** [review]

File: modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp (hunk 1 above)
Comment (ResearchDaniel) [reply modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp]: Fixed

Comment (petersteneteg) [reply modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp]: Still no change though?

**Comment by ResearchDaniel** [comment]
@petersteneteg Forgot to push, only commited :D

**Review by petersteneteg — APPROVED** [review]

**Pull request merged**